### DISCUSSION TITLE
weibo 活动签到兑换码

### DISCUSSION THREAD

**Discussion by dontdot** [Ljzd-PRO/nonebot-plugin-mystool#269] [General]
[yindan1/genshinhelper2](https://github.com/y1ndan/genshinhelper2)，用了genshinhelper2仓库里`weibo`代码做修改可以成功获取到3个兑换码，可以的话，等到原神或者星铁超话有新的活动完成测试就可以pr了，只用了获取兑换码功能，签到暂时没去弄

**Comment by dontdot** [comment]
我试了下添加weibo签到部分进去试运行，经常几天就失效了，我想问下有什么办法去每次签到完后更新cookie吗

**Reply by Ljzd-PRO** [reply]
从请求返回的Cookie中获取？或者网上搜一下看看有没有人有发现微博的API
